bootstrap/kubeadm/internal/controllers/kubeadmconfig_controller.go

```diff
@@ -249,6 +249,7 @@ func (r *KubeadmConfigReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 
 func (r *KubeadmConfigReconciler) reconcile(ctx context.Context, scope *Scope, cluster *clusterv1.Cluster, config *bootstrapv1.KubeadmConfig, configOwner *bsutil.ConfigOwner) (ctrl.Result, error) {
 	log := ctrl.LoggerFrom(ctx)
+	log.Info("Reconciling KubeadmConfig")
```

Comment:
This was added in controller-runtime recently so we don't have to add it on every reconciler: https://github.com/kubernetes-sigs/controller-runtime/pull/2419

Reply:
So we're expecting this to land in controller runtime 0.16?

Reply:
It is part of CR v0.16 correct!

To make this more useful in production (it's just on log-level 5) I will propose an endpoint soon which will make it possible to change log-levels during runtime.

Reply:
Cool - so we'll benefit from this in CAPI v1.6.0. I'm fine with including it for now in the CAPI logs if we want to. We'll have to take it out of other places with the release of v1.6.0 anyway.

Reply:
Let's keep as-is since this improvement will come anyhow. But you could add the requirement to an issue if you have one – something like "Raise log level of controller-runtime's `Reconciling` message and ensure it shows up in both production and unit test logs".

Reply:
The plan is not to raise the log level, the plan is that you can dynamically set log-level 5 at runtime and then see this log if you need to